### ISSUE TITLE
Wrap our cosmos config in <StrictMode> so all of our components are ready for Concurrent mode

### ISSUE THREAD

**Issue by brianespinosa** [Espressive/cascara#33]
Everything in Cascara needs to be ready for the future of React. We should wrap our Cosmos development environments in [`<StrictMode>`](https://reactjs.org/docs/strict-mode.html) so that we are sure to get any additional development runtime errors to guide us down the right path.

**Comment by brianespinosa** [comment]
This is going to be blocked by https://github.com/react-cosmos/react-cosmos/issues/1249#issue-675317311

**Comment by brianespinosa** [comment]
Starting on this now.

**Comment by brianespinosa** [comment]
This will be resolved in #59 but another issue was also opened #60 so we can be able to define a single global fixture config without having to place one in all of our /src folders.
